bucket/zoom.json

```
@@ -0,0 +1,31 @@
+{
+    "version": "4.6.7",
+    "description": "Video and audio conferencing, chat, and webinars",
+    "homepage": "https://zoom.com.cn/",
```

Chinese homepage seems like great idea.
```suggestion
    "homepage": "https://zoom.us",
```